temporal-testing-junit4/build.gradle

```diff
@@ -0,0 +1,165 @@
+    dependsOn 'googleJavaFormat'
+    options.encoding = 'UTF-8'
+    options.compilerArgs << '-Xlint:none' << '-Xlint:deprecation' << '-Werror'
+}
+
+compileTestJava {
+    options.encoding = 'UTF-8'
+    options.compilerArgs << '-Xlint:none' << '-Xlint:deprecation' << '-Werror'
+}
+
+if (JavaVersion.current().isJava8Compatible()) {
+    allprojects {
+        tasks.withType(Javadoc) {
+            options.addStringOption('Xdoclint:none', '-quiet')
+        }
+    }
+}
+
+javadoc {
+    options.encoding = 'UTF-8'
+    if (JavaVersion.current().isJava9Compatible()) {
+        options.addBooleanOption('html5', true)
+    }
+}
+
+task sourceJar(type: Jar) {
+    from sourceSets.main.allSource
+    classifier "sources"
+}
+
+test {
+    dependsOn 'checkLicenseMain'
+    testLogging {
+        events 'passed', 'skipped', 'failed'
+        exceptionFormat 'full'
+        // Uncomment the following line if you want to see test logs in gradlew run.
+        showStandardStreams true
+    }
+    forkEvery = 1
+    maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            from components.java
+            versionMapping {
+                usage('java-api') {
+                    fromResolutionOf('runtimeClasspath')
+                }
+                usage('java-runtime') {
+                    fromResolutionResult()
+                }
+            }
+            pom {
+                name = 'Temporal Java SDK testing for JUnit 4'
+                packaging = 'jar'
+                // optionally artifactId can be defined here
+                description = 'Temporal Workflow Java SDK testing, JUnit 4.x integration'
+                url = 'https://github.com/temporalio/temporal-java-sdk'
+
+                scm {
+                    connection = 'scm:git@github.com:temporalio/temporal-java-sdk.git'
+                    developerConnection = 'scm:git@github.com:temporalio/temporal-java-sdk.git'
+                    url = 'https://github.com/temporalio/temporal-java-sdk.git'
+                }
+
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+
+                developers {
+                    developer {
+                        id = 'mfateev'
+                        name = 'Maxim Fateev'
+                        email = 'maxim@temporal.io'
+                    }
+                    developer {
+                        id = 'samarabbas'
+                        name = 'Samar Abbas'
+                        email = 'samar@temporal.io'
+                    }
+                }
+            }
+        }
+
+    }
+
+    signing {
+        sign publishing.publications.mavenJava
+    }
+
+    // Uncomment to test local publishing and comment nexusPublishing
```

Comment:
Nit. Let's remove this as we an simply run `/gradlew publishToMavenLocal` without having to change anything.